core/controllers/base.py

```diff
@@ -129,6 +129,9 @@ class BaseHandler(webapp2.RequestHandler):
 
     # What format the get method returns when exception raised, json or html.
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_HTML
+    PUT_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
```

Comment:
The canonical order is GET, POST, PUT, DELETE -- let's follow that.

Reply:
done